backend/httpclient/max_bytes_reader_test.go

```diff
@@ -28,13 +28,13 @@ func TestMaxBytesReader(t *testing.T) {
 
 			bodyBytes, err := io.ReadAll(readCloser)
 			if err != nil {
-				require.EqualError(t, tc.err, err.Error())
+				require.EqualError(t, err, tc.err.Error())
 			} else {
 				require.NoError(t, tc.err)
 			}
 
-			require.Len(t, bodyBytes, tc.bodyLength)
-			require.Equal(t, string(bodyBytes), tc.body)
+			require.Len(t, bodyBytes, tc.expectedBodyLength)
+			require.Equal(t, tc.expectedBody, string(bodyBytes))
```

Comment:
I replaced the expected and the actual values so the failed test message can help me with what's expected and what's actual value.